appear/src/main/java/com/arcbees/gquery/appear/Appear.gwt.xml

```diff
@@ -0,0 +1,3 @@
+<module>
+    <inherits name='com.google.gwt.query.Query'/>
```

Comment:
Nope. I think it's because gwtquery includes `com.googlecode.gwtquery.client` and `Observe` is in `com.googlecode.gwtquery.client.plugins`. There's no gwt.xml file in the observe-plugni